LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceRegistry.cs

```diff
@@ -128,6 +132,8 @@ public ILoRaDeviceRequestQueue GetLoRaRequestQueue(LoRaRequest request)
         {
             if (request is null) throw new ArgumentNullException(nameof(request));
 
+            this.deviceLoadRequests?.Add(1);
```

Comment:
It seems a bit unintuitive to have a `DeviceLoadRequest` inside the `GetLoRaRequestQueue`. If I see it correctly the actual loading is done in `DeviceLoaderSynchronizer.LoadAsync`, maybe we should track it farther down in the stack, at a different location in the `LoRaDeviceRegistry`?

If the reason for the metrics being raised in different call sites is that we didn't encapsulate the caching methods (and we have to manually query the cache before issuing a search, therefore have to raise the metric on the call site), then we could also reconsider the semantics of `DeviceLoadRequest` and track the number of requests/queries that did hit the cache. The total number of load requests will always be the sum of the number of cache hits and cache misses.

Reply:
As per discussion on Teams, I pushed 34d1ea22b33a39afd68e632b41b44b00d1c1a3ca where the `DeviceLoadRequests` metric is used to count requests made against the Lora device API service and not the cache hits. The `deviceLoadRequests are now kept track of in the `LoRaDeviceAPIService.cs`. Let me know if you have further comments about this.